manifests/pnp4nagios/config.pp

```diff
@@ -0,0 +1,77 @@
+class icinga2::pnp4nagios::config inherits icinga2::pnp4nagios::params {
+
+  $monitoring_type = 'icinga'
+  $log_type = 'syslog'
+  $debug_lvl = 1
+  $system_date = generate('/usr/bin/date', '+%Z')
```

Comment:
why are all these variables local? why are they not class parameters? the user should be able to change these values if they want to.